docs/skopeo-sync.1.md

```diff
@@ -0,0 +1,80 @@
+% skopeo-sync(1)
+
+## NAME
+skopeo\-sync - Copy images from one or more repositories to a user specified destination.
+
+## SYNOPSIS
+**skopeo sync** [**--source-yaml**] _source_ _destination_
+
+## DESCRIPTION
+Copy images from one or more repositories to a user specified destination.
+
+Useful to keep in sync with a local container registry mirror. It can also be used to populate registries running inside of air-gapped environments.
+
+_source_ can be:
+ - a repository hosted on a container registry (eg: docker://registry.example.com/busybox)
+ - a local directory (eg: dir:/media/usb/).
+ - (when **--source-yaml** is specified) a YAML file with a list of source images from different container registries.
```

Comment:
I think we should be very verbose that the yaml file works only with the `docker://` transport.